resources/moneropedia/miners.md

```diff
@@ -7,14 +7,12 @@ summary: "the process of cryptographically computing a mathematical proof for a
 
 ### The Basics
 
-Mining is the distributed process of confirming transactions on the public ledger of all transactions, aka @blockchain.  Monero nodes use the block chain to distinguish legitimate transactions from attempts to re-spend coins that have already been spent elsewhere. 
-
-Monero is powered strictly by Proof of Work. It employs a mining algorithm that has the potential to be efficiently tasked to billions of existing devices (any modern x86 CPU and many GPU's). Monero uses the @CryptoNight Proof of Work (@PoW) algorithm, which is designed for use in ordinary CPUs and GPUs.
-
-The smart mining feature allows transparent CPU mining on the user's computer, far from the de facto centralization of mining farms and pool mining, pursuing Satoshi Nakamoto's original vision of a true P2P currency. 
+{{ page.summary | capitalize }}.
 
-As of June 2017, botnets made of hacked computers account for a not-insignificant portion of the miners on the Monero network.  This is due to the profitability of mining on CPUs via the @CryptoNight algorithm.
+Mining is the distributed process of confirming transactions on the public ledger of all transactions, aka @blockchain.  Monero nodes use the blockchain to distinguish legitimate transactions from attempts to re-spend coins that have already been spent elsewhere. 
 
+Monero is powered strictly by Proof of Work. It employs a mining algorithm that has the potential to be efficiently tasked to billions of existing devices (any modern x86 CPU and many GPUs). Monero uses the CryptoNight Proof of Work (PoW) algorithm, which is designed for use in ordinary CPUs and GPUs.
 
+The smart mining feature allows transparent CPU mining on the user's computer, far from the de facto centralization of mining farms and pool mining, pursuing Satoshi Nakamoto's original vision of a true P2P currency. 
 
-{{ page.summary | capitalize }}.
+As of June 2017, botnets made of hacked computers account for a not-insignificant portion of the miners on the Monero network.  This is due to the profitability of mining on CPUs via the CryptoNight algorithm.
```

Comment:
What's the source of this information? I don't believe even if this is true that it's good to include in the Entry for mining. We don't want to promote the use of botnets to mine, but we can't stop people from doing so.

Reply:
@QuickBASIC I echo these comments. This should not be in the entry for mining, especially without a source.

Reply:
I didn't add that paragraph, should I remove it anyway?

Reply:
Weird. It probably showed in the did because of moving text a around. Yes, please.

Reply:
Removed, PTAL.